src/twisted/web/http.py

```diff
@@ -2296,6 +2300,17 @@ def timeOut(self, value):
         self._channel.timeOut = value
 
 
+    @property
+    def callLater(self):
+        return self._channel.callLater
```

Comment:
For the sake of clarity:  `HTTPChannel` inherits from `TimeoutMixin`, which defines a `callLater` method that just imports the reactor and calls *its* `callLater`.  So patching this patches *that*.